Bottom: 853fafaea0a7c48043a577e2330cc7dabb652f67
Top:    ad2d146c9ab3b6d75023609ee1ce68b327e3f5a8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 23:15:19 +0000

Temp


---

diff --git a/src/polygon.c b/src/polygon.c
index 7a47ae8..e9cac40 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2385,6 +2385,8 @@ POLYAREA *board_outline_poly (bool include_holes)
   GList *pieces_to_delete = NULL;
   bool any_pieces_kept = false;
 
+  return NULL;
+
 #define BLOAT_WORLD MIL_TO_COORD (10)
 
   whole_world = RectPoly (-BLOAT_WORLD, BLOAT_WORLD + PCB->MaxWidth,
